src/agent/src/rpc.rs

```diff
@@ -106,6 +106,8 @@ use kata_types::k8s;
 
 pub const CONTAINER_BASE: &str = "/run/kata-containers";
 const MODPROBE_PATH: &str = "/sbin/modprobe";
+const INIT_TRUSTED_STORAGE: &str = "/usr/bin/kata-init-trusted-storage";
+const TRUSTED_IMAGE_STORAGE_DEVICE: &str = "/dev/trusted_store";
```

Comment:
to keep it coherent, shouldn't it be `TRUSTED_STORAGE_DEVICE` (without IMAGE)?

Reply:
This device is used for image storage, but the script `kata-init-trusted-storage` serves additional scenarios beyond just  trusted image storage.